### PR TITLE
add feb2022 road and trails update

### PR DESCRIPTION
**Pull request by gregbunce**
including the two network datasets, a bunch of locators, and the ng911 database.

### CODE REVIEW

File: _posts/2022-02-10-utah-sgid-statewide-roads-data-layer-updates-02102022.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+New roads were added and road names and address ranges were updated for the following counties:
+
+- **Davis County:** Obtained roads data on 01/14/2021. Previous update was on 01/12/2022.
+- **Salt Lake County:** Obtained roads data on 01/31/2021. Previous update was on 01/12/2022.
+- **Summit County:** Obtained roads data on 01/14/2021. Previous update was on 12/13/2021.
+- **Utah County:** Obtained roads data on 01/14/2021. Previous update was on 12/13/2021.
+- **Wasatch County:** Obtained roads data on 02/03/2021. Previous update was on 11/09/2021.
+- **Washington County:** Obtained roads data on 01/14/2021. Previous update was on 01/12/2022.
+- **Weber County:** Obtained roads data on 01/14/2021. Previous update was on 01/12/2022.
```
Comment (stdavis) [inline _posts/2022-02-10-utah-sgid-statewide-roads-data-layer-updates-02102022.md:29]: I think that some of these dates may be reversed...

**Review by steveoh — DISMISSED** [review]
When did the lrs changes happen?

**Comment by gregbunce** [comment]
that was probably a year ago now.  i could remove that sentence next month. it's probably time.

**Comment by gregbunce** [comment]
oops, thanks, scott.  i'll fix that.

**Comment by gregbunce** [comment]
@stdavis i think they're correct.  they look funny, but they're correct.

**Comment by gregbunce** [comment]
ahh.....  jeez.  i'm struggling today.  thanks.

**Comment by gregbunce** [comment]
thanks for catching that.  if it looks good to you, it can be merged.  thanks again.

**Review by steveoh — APPROVED** [review]

**Review by stdavis — APPROVED** [review]

**Pull request merged**